lib/matplotlib/figure.py

```diff
@@ -2018,6 +2021,158 @@ def tight_layout(self, renderer=None, pad=1.08, h_pad=None, w_pad=None,
             pad=pad, h_pad=h_pad, w_pad=w_pad, rect=rect)
         self.subplots_adjust(**kwargs)
 
+    def align_ylabels(self, axs=None, renderer=None):
+        """
+        Align the ylabels of subplots in this figure.
+
+        Choose the left-most label to align to.  Axes with the same subplot
+        column are aligned.
+
+        Parameters:
```

Comment:
I'll channel the numpydoc police here.

Also seems normal to define align_xlabels first?